guide/english/security/social-engineering/index.md

```diff
@@ -12,6 +12,12 @@ Social Engineering is the art of gaining access to a secured system or resource
 * Charming and persuasive
 * Patient and observant
 * Adept at predicting human behavior based on exploiting the human need to be helpful, curious, greedy and vain
```

Comment:
Hi
Not able to edit

Reply:
click on the pen icon on top right corner